backend/src/main/kotlin/org/icpclive/service/ICPCScoreboardService.kt

```diff
@@ -126,7 +126,7 @@ abstract class ICPCScoreboardService(optimismLevel: OptimismLevel) {
                     (okRun ?: runsBeforeFirstOk.lastOrNull())?.time
                 ).also {
                     if (it.score > 0) {
-                        solved++
+                        solved += it.score
```

Comment:
It's not up to me to judge, but solved is semantically the number of solved tasks – not the overall score.
May break some things somewhere

Reply:
Also bear in mind that ScoreboardService is considered crutial because depending on it we might show wrong scoreboard and we can't allow for it.

Reply:
I don't see any indications that `totalScore` is the numbers of solved problems. Sorting has been fixed, now it is by the one who scored more points in total. No breakdowns occurred, on the contrary, now the amount in the table is displayed correctly, as well as on https://vk.com/wall-71991592_39972

![image](https://user-images.githubusercontent.com/115399684/201541370-9de91458-38a5-4941-ac6b-d04e5092f627.png)
![image](https://user-images.githubusercontent.com/115399684/201541147-a92588db-2694-4a3b-a029-579b3a7bf7f4.png)

I understand what it's about, but I don't see any reason to create a new variable for total team score and do another sort

Reply:
Even if you look at the use of this variable, it is responsible for sorting, and the meaning has not changed in the conditions. Checks only determine whether the user has solved at least one task - and nothing changes in the situation with Score.

![image](https://user-images.githubusercontent.com/115399684/201541733-46dbd222-91f2-4d61-b199-22da84abb998.png)

Reply:
Fair enough.
But bear in mind that score can sometimes be a rational number. Like in the Huawei ICPC Challenge.
@kunyavskiy what do you say?

Reply:
@irdkwmnsb rational numbers in scores is not just a problem with `totalScore` and with the fact that `score` is Int, it's also an overlay design problem if score is more than 9. See more on the screenshot in Queue. I made an experimental support for Float's here: https://github.com/icpc/live-v3/pull/81/commits/6ebcc3a1e881d9c2ee0d42216cbf327ecd7492ea.

![image](https://user-images.githubusercontent.com/115399684/201611647-4fd3c19f-a22d-4be2-a29c-d766c6f5762e.png)

Compatibility with integers remained.